sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/TokenRefresher.java

```diff
@@ -12,5 +12,5 @@
      * Asynchronous call to fetch a fresh token
      * @return Wrapper for asynchronous call
      */
-    Future<String> getFetchTokenFuture();
+    Mono<String> getFetchTokenNext();
```

Comment:
This is a public interface. IMHO, we'd need a better name. getTokenAsync?

Reply:
Updated to getTokenAsync